docs/rpc-endpoints.md

```diff
@@ -503,6 +503,16 @@ data.
 
 This will return 404 if the block does not exist.
 
+### GET /v3/blocks/height/[Block Height]
+
+Fetch a Nakamoto block given its block height.  This returns the raw block
+data.
+
+This will return 404 if the block does not exist.
+
+This endpoint also accepts a querystring parameter `?tip=` which when supplied will return the
```

Comment:
It might be worth mentioning that the value of this parameter is that it enables querying sibling blocks.